cli/azd/pkg/account/subscriptions_manager.go

```diff
@@ -160,8 +160,9 @@ func (m *SubscriptionsManager) ListSubscriptions(ctx context.Context) ([]Subscri
 	ctx, span := tracing.Start(ctx, events.AccountSubscriptionsListEvent)
 	defer span.EndWithStatus(err)
 
-	stop := m.msg.ShowProgress(ctx, "Retrieving subscriptions...")
-	defer stop()
+	msg := "Retrieving subscriptions..."
+	m.console.ShowSpinner(ctx, msg, input.Step)
+	defer m.console.StopSpinner(ctx, "", input.StepDone)
```

Comment:
Do we need to make these sets of changes now? Feels like we're starting to bloat the PR.

Reply:
I'd hold off on this since whatever new progress behavior we are going with should also apply here.

Reply:
This is a redundant implementation to wrap the console. We don't need to maintain it